lib/bibliothecary/configuration.rb

```diff
@@ -18,6 +19,7 @@ def initialize
       @mix_parser_host      = 'https://mix.libraries.io'
       @gradle_parser_host   = 'https://gradle-parser.libraries.io'
       @yarn_parser_host     = 'https://yarn-parser.libraries.io'
+      @conda_parser_host    = 'https://conda.libraries.io'
```

Comment:
Working on getting some of these timeout errors to go away - so this can't merge until some internal things are settled.